src/main/java/org/folio/consortia/domain/repository/UserTenantRepository.java

```diff
@@ -0,0 +1,19 @@
+package org.folio.consortia.domain.repository;
+
+import org.folio.consortia.domain.entity.UserTenantEntity;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+import java.util.UUID;
+
+@Repository
+public interface UserTenantRepository extends JpaRepository<UserTenantEntity, UUID> {
```

Comment:
I think this contract should be simpified. Maybe something like this(we will have only 2 methods):
List<UserTenantEntity> findByUserIdAndTenantId(String userId, String tenantId);
List<UserTenantEntity> findByUsernameAndTenantId(String username, String tenantId);

So in this case tenantId will be required param for us, API clients should not think is username can be required or not. 
Also we will have unified contract, @okolawole-ebsco what do you think?

Reply:
And also we can fetch tenantId from X-OKAPI_TENANT to simplify REST contract, API cliens should provide only userId or username as request params

Reply:
Talked with @okolawole-ebsco 
Contract should be like this:
```
List<UserTenantEntity> findByUserId(UUID userId);
UserTenantEntity findByUsernameAndTenantId(String username, String tenantId);
```